src/applications/ezr/utils/helpers/form-config.js

```diff
@@ -146,7 +146,7 @@ export function includeAgentOrangeExposureDates(formData) {
 export function includeOtherExposureDates(formData) {
   const { 'view:otherToxicExposures': otherToxicExposures = {} } = formData;
   const exposures = Object.values(otherToxicExposures);
-  return exposures.some(o => o) && includeTeraInformation(formData);
+  return exposures.includes(true) && includeTeraInformation(formData);
```

Comment:
May not be a bad idea to callout in the PR notes that some implementation was changed, since this is different than before. It has better test coverage, so that's certainly a 👍 , but just wanted to callout it's not all spec changes.

Reply:
Yes, I'll update the description of the PR to include (no pun intended) the change to this piece of code!